local-cli/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
```

Comment:
I believe this is just going to be:

module.exports = [ 
    require('./runWindows/runWindows)
];